### ISSUE TITLE
question

### ISSUE THREAD

**Issue by vegarringdal** [Wakanda-Packages/wakanda-widgets#68]
I see in the #this widget was verified for wakanda versions 8 -by vegar ringdal"
Most my testing is in wak 9.

and if you guys could point addon to my widget to commit 7a668cd3a71d04d09dc01b88fed1578e5b16cce6
then that would be great, thx

**Comment by rmello4d** [comment]
Vegar, Just update the engines value in your package JSON, you can use 
engines : { wakanda : "9" }    or 
engines : { wakanda : [ "9", "10"]  }    or

about your commit, can you send us your pull request ? we are automating the process, very soon will be easier and the server now is updated every 15 mins

**Comment by vegarringdal** [comment]
ok, Ill send a new pull request.

btw just tested wak10 beta today again.
is there a lot of bugs that you are fixing in the custom login api?
because I have some weird bugs in my application: https://github.com/vegarringdal/PTOC


from what I see it pretty much kills/breaks my input widget
https://github.com/vegarringdal/InputText

some places it looks weird in studo, some places I cant get it to work in components, etc etc

**Comment by vegarringdal** [comment]
did some more testing, looks like how I use th value propertie needs to changed in mine, I wrote the class.attribute like old widegt did it, not the datasource link does it too, so that might be the issue, so my app update just wont go that well in the start, hehe

**Comment by rmello4d** [comment]
I see :) It will be nice if you could give me a status of your PTOC solution on wakanda 10 ...

**Comment by vegarringdal** [comment]
looks like I solved most issues..
Had to delete a few grids some places, deleting them and adding them fixed it, might haveto do with the way studio was edited like you said in a earlier post.
There are a few css bugs in studio and browser, in studio my input widget does not get correct classes for some reason, but if I put it in a webcomponent and put that on the page it displays correctly, created a bug raport for that one.
There is also a blue border comming everywhere in the broswer on buttons and inputs/editable divs everywhere in chrome atleast, looked like matrix css what doing it, think welsh created a bug raport for that one.

Can highlight text in internet explorer now in wak 10, that was nice.

There have been a issue with websockets and internet explorer in wak 9, for some reason the onclose event didnt work sometimes, and websockets locked up, so if you code was dependent on getting an answer, created a bug raport on this, hope it gets solved if its still an issue in wak10, havent check it much yet, I just disabled websockets if it was IE browser, and I never had any issues with websockets after.

Dunno if I ever get to use my PTOC solution in production, but its a good way to learn, did a huge rebuild of the entire clientside last months, need to remove some old inputwidgets, just replaced it where there was numbers and dates, since I can control how decimal/numbers and dateformat is displayed easier in my widget.
Added a event when page is loaded, so I can set a what setting to use by code instead of in studio.
This way I can set a global for app, or a per user (havent created that in app yet)
